packages/cli/lib/utils.js

```diff
@@ -5,14 +5,19 @@
 
 'use strict';
 
+const debug = require('../lib/debug')('utils');
 const fs = require('fs');
+const path = require('path');
 const util = require('util');
 const regenerate = require('regenerate');
 const _ = require('lodash');
 const pascalCase = require('change-case').pascalCase;
+const promisify = require('util.promisify/implementation');
```

Comment:
Oh, sorry for a misleading comment. Your code here will use the user-land promisify implementation even on Node.js 8.x where a native version is available.

Here is a better solution:

```js
const promisify = util.promisify || require('util.promisify/implementation');
// ...

exports.promisify = promisify;
```  

(I am proposing to export the promisify helper so that I can easily access it in my https://github.com/strongloop/loopback-next/pull/848 too).